WSL/setup/environment.md

```diff
@@ -44,6 +44,8 @@ Once the process of installing your Linux distribution with WSL is complete, ope
 
 - This **User Name** and **Password** is specific to each separate Linux distribution that you install and has no bearing on your Windows user name.
 
+- Please note that whilst entering the **Password**, nothing will appear on screen. This is called **blind typing**. You won't see what you are typing, this is completly **normal**.
```

Comment:
nits: completly fix typo and unbold the "blind typing" and "normal"

Thank-you!

Reply:
Thanks for the corrections! They have been resolved.